src/pages/LNInvoice.tsx

```diff
@@ -21,63 +27,36 @@ const LNInvoice = () => {
     let stopPolling = false;
     //const shouldStopPolling = () => stopPolling;
 
-    const getInvoice=()=>{
-      fetch('https://cybermart.com.jm/api/v1/payments/',
-      {
-          headers : { 
-              'Content-Type': 'application/json',
-              'Accept': 'application/json',
-              'X-Api-Key':'5cc6e1a550ac465597c4d316fe8e08c0'
-          },
-          body: JSON.stringify(
-              {
-                  "out": false, 
-                  "amount": subtotal, 
-                  "memo": Object.values(reciept).join(","), 
-                  "webhook": "google.com"
-              }),
-          method: 'post'
-      })
-      .then(function(response){
-        return response.json();
-      })
-      .then(function(myJson) {
+    const getInvoice = async ()=>{
+      const { invoice, params, successAction, validatePreimage } =
+          await requestInvoice({
+            lnUrlOrAddress: 'alivesession77@walletofsatoshi.com',//'frazras@pay.bitcoinjungle.app',
+            tokens: subtotal as Satoshis
+          });
+       if ((window as any).webln) {
```

Comment:
checking if alby is installed

Reply:
when I deactivate alby it justs stays on the processing gif. think this'd be a good error check, try catch if alby's installed otherwise redirect to alby